configure

```diff
@@ -213,6 +218,9 @@ elif isbsd ; then
 elif issolaris; then
 	echo "CFLAGS+=-DIS_SOLARIS -D__EXTENSIONS__" >> config.mak
 	echo "SOCKET_LIBS=-lsocket -lnsl" >> config.mak
+elif ishaiku ; then
+	echo LIBDL=>>config.mak
+	echo "CFLAGS+=-DIS_HAIKU" >> config.mak
```

Comment:
Maybe add this?
```diff
+	echo "SOCKET_LIBS=-lnetwork" >> config.mak
```

Reply:
maybe? does it fix it or not ?

Reply:
> maybe? does it fix it or not ?

Doesn't fix, but on Haiku 32bit with gcc2 we use -lsocket, for gcc8 we have to use -lnetwork, so if this is a requirement I think it should be included?

Reply:
why does the choice of gcc affect in which library stuff is being put? that sounds illogical.
which functions that we require/hook reside in libsocket/libnetwork?

Reply:
Not sure (not a developer here, and don't know the ins and outs for the Haiku core), but could be that -lsocket is kept for backwards compatibility for BeOS apps/libraries

Reply:
PS, thought that the lines preceding this would give a clue that it could be needed?
```
echo "SOCKET_LIBS=-lsocket -lnsl" >> config.mak
```